packages/sit-onyx/src/components/OnyxLink/OnyxLink.vue

```diff
@@ -29,15 +33,49 @@ defineSlots<{
     @click="emit('click')"
   >
     <slot></slot>
+    <OnyxIcon
+      v-if="!props.hideExternalIcon && isExternalLink(props.href)"
+      class="onyx-link__icon"
+      :icon="arrowSmallUpRight"
+      size="16px"
+    />
   </a>
 </template>
 
 <style lang="scss">
 .onyx-link {
   text-decoration: underline;
   color: var(--onyx-color-text-icons-primary-intense);
+  border: 1px solid transparent;
 
   // other styles like size, font family etc. should be inherited
   // so we don't define them here
+
+  &:hover {
+    color: var(--onyx-color-text-icons-primary-medium);
+  }
+
+  &:focus-visible {
+    color: var(--onyx-color-text-icons-primary-intense);
+    border-radius: var(--onyx-radius-sm);
+    border-color: var(--onyx-color-base-primary-300);
+    outline: none;
```

Comment:
Semantically and technical (doesn't take any space) `outline` is preferable here. `outline` will adhere to the `border-radius`

Reply:
<img width="344" alt="image" src="https://github.com/SchwarzIT/onyx/assets/67898185/40c5854c-edae-420b-ab9c-1107e3b71f52">

Do you know how to prevent the outline to not have this break at the icon?

Reply:
Is it already fixed? I am unable to reproduce the issue

Reply:
this works on all browsers for me
```scss
  text-decoration: underline;
  color: var(--onyx-color-text-icons-primary-intense);
  border-radius: var(--onyx-radius-sm);
  outline: 1px solid transparent;

  // other styles like size, font family etc. should be inherited
  // so we don't define them here

  &:hover {
    color: var(--onyx-color-text-icons-primary-medium);
  }

  &:focus-visible {
    color: var(--onyx-color-text-icons-primary-intense);
    outline-color: var(--onyx-color-base-primary-300);
  }
```

Reply:
I needed to add `display: inline-block;`, It works now with the outline